code/script-0212.js

```diff
@@ -0,0 +1,274 @@
+  );
+  saladBtn.addEventListener("click", (event) =>
+    setTimeout(() => showSubOptions(event), 1000)
+  );
+};
+
+//
+//Show user's main choice
+const showUserMainChoice = (event) => {
+  showMessage(`${event.target.value}`, `user`);
+};
+
+//
+//Show bot's reply and sub options
+const showSubOptions = (event) => {
+  if (event.target.value === "Pizza") {
+    inputWrapper.innerHTML = `
+      <select id="select" onchange="bubbleAudio.play()">
+        <option value="" disabled selected>--SELECT ONE ITEM--</option>
+        <option value="Margherita" name="pizza-Margherita" id="margherita">Margherita</option>
+        <option value="Funghi" name="pizza-Funghi" id="funghi">Funghi</option>
+        <option value="Pepperoni" name="pizza-Pepperoni" id="pepperoni">Pepperoni</option>
+      </select>`;
+  } else if (event.target.value === "Pasta") {
+    inputWrapper.innerHTML = `
+      <select id="select" onchange="bubbleAudio.play()">
+      	<option value="" disabled selected>--SELECT ONE ITEM--</option>
+        <option value="Spaghetti Carbonara" name="Spaghetti-Carbonara" id="carbonara">Spaghetti Carbonara</option>
+        <option value="Fussili Pesto" name="Fussili-Pesto" id="pesto">Fussili Pesto</option>
+        <option value="Spaghetti-Bolognese" name="Spaghetti-Bolognese" id="bolognese">Spaghetti Bolognese</option>
+      </select>`;
+  } else {
+    inputWrapper.innerHTML = `
+      <select id="select" onchange="bubbleAudio.play()">
+        <option value="" disabled selected>--SELECT ONE ITEM--</option>
+        <option value="Ceaser Salad" name="Ceaser-Salad" id="ceaser">Ceaser Salad</option>
+        <option value="Chef Salad" name="Chef-Salad" id="chef">Chef Salad</option>
+        <option value="Greek Salad" name="Greek-Salad" id="greek">Greek Salad</option>
+    	</select>`;
+  }
+
+  showMessage(
+    `You ordered ${event.target.value}! Choose one from our menu.`,
+    `bot`
+  );
+
+  const subOption = document.getElementById("select");
+
+  subOption.addEventListener("change", showUserSubChoice);
+  subOption.addEventListener("change", (event) =>
+    setTimeout(() => showSizeOptions(event), 1000)
+  );
+};
+
+const showUserSubChoice = (event) => {
+  showMessage(`${event.target.value}`, `user`);
+};
+
+const showSizeOptions = (event) => {
+  showMessage(
+    `You ordered ${event.target.value}! Which size would you like to have?`,
+    `bot`
+  );
+
+  inputWrapper.innerHTML = `
+  <button id="bigBtn" value="Big" onclick="bubbleAudio.play()">Big</button>
+  <button id="smallBtn" value="Small" onclick="bubbleAudio.play()">Small</button>
+  `;
+
+  const bigBtn = document.getElementById("bigBtn");
+  const smallBtn = document.getElementById("smallBtn");
+
+  bigBtn.addEventListener("click", userConfirm);
+  smallBtn.addEventListener("click", userConfirm);
+};
+
+//
+//
+//Choosing the size of the food
+//+bot's response
+//+ cofirmation button
+
+const userConfirm = (event) => {
```

Comment:
To be consistent with naming, perhaps this function could be named askForConfirmation?